### PR TITLE
Notes tab

### PR DESCRIPTION
**Pull request by seonaidmore**
I've added the notes tab plus tried to remove some notes, but some are within a function and defined as "subheading", can't see an alternative except replacing the text with paste0(""), what do you think?

### CODE REVIEW

**Review by RosalynLP — CHANGES_REQUESTED** [review]
It's looking really good!
 I've removed the stray bits of text and also just fixed a couple of typos. Before merging we need to:

- Fix the links down the left hand side of the notes tab
- Merge master
-
I'll try and merge master in now as it could be a bit messy.

File: shiny_app/global.R [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Uncomment the line below if you are running the app locally from a different directory to the one given
 ## Make sure it is commented out before deploying the app
-#setwd("/conf/PHSCOVID19_Analysis/COVID-19-Publication-Dashboard/shiny_app")
+setwd("/conf/PHSCOVID19_Analysis/COVID-19-Publication-Dashboard/shiny_app")
```
Comment (RosalynLP) [inline shiny_app/global.R]: Have commented this out again

**Review by RosalynLP — APPROVED** [review]
Looks great, just recommented the setwd as it won't deploy with that uncommented

File: shiny_app/global.R (hunk 1 above)
Comment (rubenv95) [reply shiny_app/global.R]: Oh yeah cool, didn't spot that, thanks

**Pull request merged**